homeassistant/components/binary_sensor/deconz.py

```diff
@@ -113,3 +115,21 @@ def device_state_attributes(self):
         if self._sensor.type in PRESENCE and self._sensor.dark is not None:
             attr[ATTR_DARK] = self._sensor.dark
         return attr
+
+    @property
+    def device(self):
+        """Description for device registry."""
+        if (self._sensor.uniqueid is not None and
```

Comment:
Can you inverse this logic to make the method a bit easier to read

```python
if (self._sensor.uniqueid is None or
        self._sensor.uniqueid.count(':') != 7):
    return None
```

Reply:
I can evaluate positive on the counter instead. Then leave return None to be at the end of the method instead.